lib/src/base-styles/_nx-tables.scss

```diff
@@ -183,6 +183,11 @@
   -ms-user-select: none;
 }
 
+.nx-cell--filter-header {
+  background-color: #ffffff;
+  border-bottom: 1px solid #e9e9e9;
```

Comment:
Looks like this should be #d1d6f0

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/e53575ee7fe3dae2abc0bf391e9af3e2321acb9f